.github/workflows/test.yml

```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -U pip setuptools wheel
-          pip install -r ./requirements.txt
+          # FIXME remove git when django-stubs is released
+          pip install -r ./requirements.txt git+https://github.com/intgr/django-stubs@integration
```

Comment:
For now I have pushed this branch, which contains all the dependant branches merged together.